src/patternfly/components/Switch/switch.scss

```diff
@@ -20,6 +25,14 @@
   --pf-c-switch__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
   --pf-c-switch__toggle--before--BorderRadius: var(--pf-c-switch__toggle--before--Width);
```

Comment:
can you change the value of this to `--pf-global--BorderRadius--lg` it will fix that bug in chrome.